### PR TITLE
Created build.sh

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Included build.sh to build form non-windows.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+if test "$OS" = "Windows_NT"
+then
+  # use .Net
+
+"./tools/nuget/nuget.exe" "install" "xunit.runner.console" "-OutputDirectory" "tools" "-ExcludeVersion" "-version" "2.0.0"
+"./tools/nuget/nuget.exe" "install" "FAKE.Core" "-OutputDirectory" "tools" "-ExcludeVersion" "-version" "4.4.2"
+"./tools/nuget/nuget.exe" "install" "SourceLink.Fake" "-OutputDirectory" "tools" "-ExcludeVersion" "-version" "1.1.0"
+packages/FAKE/tools/FAKE.exe $@ --fsiargs -d:MONO build.fsx
```
Comment (shiftkey) [inline build.sh]: Do we need to pass in the `MONO` flag here?

**Comment by shiftkey** [comment]
I'll take this in and then rework my stuff in #924 to be x-plat friendly. Just that one question.

File: build.sh (hunk 1 above)
Comment (naveensrinivasan) [inline build.sh]: It is for conditional compilation to `MONO`. I don't think so.

**Comment by naveensrinivasan** [comment]
@shiftkey If you are going to pull this then can I  close this PR?

**Comment by shiftkey** [comment]
@naveensrinivasan sorry, I should have been clearer. As this is new code (around an area we don't currently support) I'm happy to take this in and then iterate on it.

File: build.sh (hunk 1 above)
Comment (shiftkey) [inline build.sh]: :thumbsup:

**Comment by naveensrinivasan** [comment]
Thanks. I was thinking we should start supporting mono, its just the question of making the builds ready for it. The code supports it. Once the build scripts are ready for non-windows we could even enable builds on https://travis-ci.org/

**Pull request merged**